sdk/identity/Azure.Identity/tests/IdentityRecordedTestBase.cs

```diff
@@ -12,6 +12,8 @@ public abstract class IdentityRecordedTestBase : RecordedTestBase<IdentityTestEn
     {
         protected IdentityRecordedTestBase(bool isAsync, RecordedTestMode? mode = default) : base(isAsync, mode)
         {
+            // TODO: enable after new KeyValue is released (after Dec 2023)
```

Comment:
Should/do we have an issue tracking this?

Reply:
there's one already - https://github.com/Azure/azure-sdk-for-net/issues/39914, I was a bit lazy and didn't add it to the comments.